src/js/containers/covid19/budgetCategories/BudgetCategoriesTableContainer.jsx

```diff
@@ -159,32 +264,29 @@ const BudgetCategoriesTableContainer = (props) => {
 
     useEffect(() => {
         setSortAndOrderCallback();
-        storeSortAndOrderObjectCallback();
-        fetchBudgetSpendingCallback();
     }, [props.type, spendingCategory]);
 
+    useEffect(() => {
+        storeSortAndOrderObjectCallback();
+    }, [sort, order]);
+
     useEffect(() => {
         // Reset to the first page
         changeCurrentPage(1);
         fetchBudgetSpendingCallback();
     }, [props.type, pageSize]);
 
-    useEffect(() => {
-        storeSortAndOrderObjectCallback();
-        fetchBudgetSpendingCallback();
-    }, [props.type, pageSize, sort, order, spendingCategory]);
-
     useEffect(() => {
         fetchBudgetSpendingCallback();
-    }, [currentPage]);
+    }, [currentPage, sortAndOrder]);
```

Comment:
Page should reset to 1 when the sort / order changes

Reply:
Fixed.